libraries/AP_Motors/AP_Motors6DOF.cpp

```diff
@@ -407,7 +409,8 @@ void AP_Motors6DOF::output_armed_stabilizing_vectored()
     float linear_out[AP_MOTORS_MAX_NUM_MOTORS]; // 3 linear DOF mix for each motor
 
     // initialize limits flags
-    limit.roll_pitch = false;
+    limit.pitch = false;
```

Comment:
pitch -> roll

Reply:
I hate being dyslexic some time!!! I went through and checked each one of those TWICE!

Reply:
Thanks mate!!! Fixed